xml/System.Net.Security/SslServerAuthenticationOptions.xml

```diff
@@ -314,7 +334,7 @@
         <ReturnType>System.Net.Security.ServerCertificateSelectionCallback</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Gets or sets the <see cref="T:System.Net.Security.ServerCertificateSelectionCallback" /> delegate responsible for the server certificate selection.</summary>
```

Comment:
There is a nice [comment](https://github.com/dotnet/runtime/blob/master/src/libraries/System.Net.Security/src/System/Net/Security/SecureChannel.cs#L636-L639) explaining the server certificate selection. Maybe it's worth mentioning.